java/code/src/com/redhat/rhn/taskomatic/core/TaskomaticDaemon.java

```diff
@@ -15,9 +15,13 @@
 package com.redhat.rhn.taskomatic.core;
 
 import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.PosixParser;
```

Comment:
Maybe you don't have the rule enabled in your IDE but `imports` should be alphabetically ordered.

Reply:
ok. i'll take care of it.